linodecli/plugins/obj.py

```diff
@@ -798,50 +819,46 @@ def call(args, context):  # pylint: disable=too-many-branches,too-many-statement
         type=str,
         help="The cluster to use for the operation",
     )
+    return parser
 
-    parsed, args = parser.parse_known_args(args)
 
-    # don't mind --no-defaults if it's there; the top-level parser already took care of it
-    try:
-        args.remove("--no-defaults")
-    except ValueError:
-        pass
+def call(
+    args: List[str], context: PluginContext
+):  # pylint: disable=too-many-branches,too-many-statements
```

Comment:
i would like to remove these pylint disables when possible, i threw them up when adding lint so we could ignore the errors while we refactored, but now hopefully our refactoring fixes these errors and they don't need to be disabled

Reply:
Thank you for the suggestion! Let me try if the linter is still reporting errors. If the errors are still there, may I keep this disables and fix them in the next PR? I would like to keep each PR small and easier for reviews.

Reply:
Unfortunately, it is still reporting the errors...